bld/namelist_files/namelist_definition.xml

```diff
@@ -2955,12 +2955,31 @@ Default: 10
 <!-- Moist Convection and Microphysics -->
 
 <entry id="deep_scheme" type="char*16" category="conv"
-       group="phys_ctl_nl" valid_values="ZM,UNICON,off,CLUBB_SGS" >
+       group="phys_ctl_nl" valid_values="ZM,UNICON,off,CLUBB_SGS,YOG" >
 Type of deep convection scheme employed.  'ZM' for Zhang-McFarlane;
-'off' for none; or 'UNICON' which doesn't distinquish shallow and deep.
+'off' for none;'YOG' for Yanni-O'Gorman; or 'UNICON' which doesn't
+distinquish shallow and deep.
 Default: 'ZM' unless using 'UNICON', 'SPCAM' or 'pbl=none'
 </entry>
 
+<entry id="run_deep_comp" type="char*16" category="conv"
+       group="phys_ctl_nl" valid_values="off,on">
+Whether or not to use both schemes for deep convection.
```

Comment:
Which "both"? One of ZM, UNICON, SPCAM or YOG?

Reply:
This has been removed in the most recent refactor.